Makefile

```diff
@@ -1,7 +1,22 @@
-all r4: .setup-cqf-ruler-r4 synthea generate-patients-r4 calculate-patients-r4
-
+MEASURE_DIRS := $(shell ls -d EXM_*)
 export SYNTHEA_DIR := synthea_output/$(shell date +%Y-%m-%dT%H%M%S)
 
+define gen_calc_pts
+	make -C $1 all;
+endef
+
+define load_all_pts
+	./load-cqf-ruler.sh $1 FHIR_VERSION=r4;
+endef
+
+all: .setup-cqf-ruler-r4 synthea
+	$(foreach dir,$(MEASURE_DIRS),$(call gen_calc_pts,$(dir)))
```

Comment:
Would it be possible to rework this so we can `make all` with different FHIR versions, either like `make all r4` or `make all-r4` or `make r4`.

Reply:
And same would go for preloading.

Reply:
I think that should be doable. I'm assuming we still want the `r4` tasks to be the default though, since I think the `stu3` stuff is mostly legacy at this point?